.vscode/settings.json

```diff
@@ -3,5 +3,8 @@
     "source.fixAll.eslint": true
   },
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "[javascriptreact]": {
```

Comment:
I wouldn't put this in this PR, always best to keep PRs focused on one thing. But it's fine for now, you can leave it in because I will actually move `settings.json` out of git and enforce linting on commit